test/adaptive_load/utility.cc

```diff
@@ -0,0 +1,46 @@
+#include "test/adaptive_load/utility.h"
+
+#include <google/protobuf/util/time_util.h>
```

Comment:
fwiw, I'm surprised check-format didn't catch this, we might want to code in a check as I think we tend to avoid
depending on proto directly; the idea is that we do so by proxy via Envoy's facilities.

Reply:
Oh yes, forgot about that. It turns out I had even already included protobuf.h.